user-guide/Advanced_Modules/DOM/DOM_actions.md

```diff
@@ -176,7 +176,7 @@ The `DomInstanceButtonDefinition` has the following properties:
 | Id | string | The ID of the button. This must be unique to this `DomBehaviorDefinition` and must be lower case. |
 | Layout | DomButtonDefinitionLayout | Contains extra properties to define how the button will be displayed (see properties table below). |
 | VisibilityCondition | IDomInstanceCondition | Condition that defines when the button will be shown. |
-| ActionDefinitionIds | List\<string> | Contains one or more IDs of actions that should be executed. |
+| ActionDefinitionIds | List\<string> | Contains the ID of the action that should be executed. Can contain multiple, but currently, only one is allowed. |
```

Comment:
Suggestion: Maybe also mention that when multiple are configured , it will result in an exception (InvalidButtonActionCombination )

Reply:
All errors are listed on the 'DomBehaviorDefinition' page.